apps/settings/test/unit/panels/screen_lock_passcode/screen_lock_passcode_test.js

```diff
@@ -16,7 +17,8 @@ suite('ScreenLockPasscode > ', function() {
       'shared_mocks/mock_navigator_moz_settings',
       'shared_mocks/mock_settings_listener',
       'unit/mock_settings_service',
-      'panels/screen_lock_passcode/screen_lock_passcode'
+      'panels/screen_lock_passcode/screen_lock_passcode',
+      'shared/passcode_helper'
```

Comment:
I would suggest to use a mock instead of a real passcode helper. A mock is easier to configure to fit the needs of our test cases.